### PR TITLE
chore: ignore deno.lock using the config

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Comment by KnorpelSenf** [comment]
Can you expand the scope of this PQ and ensure Deno 1.29 compatibility? This includes TS 4.9.3 support.

File: deno.jsonc [hunk 1]
```diff
@@ -32,5 +32,6 @@
                 "./bundling/bundles"
             ]
         }
-    }
+    },
+    "lock": false
```
Comment (KnorpelSenf) [inline deno.jsonc:36]: Can you make this the first line in the config file? It gets a bit lost here

**Comment by KnorpelSenf** [comment]
Pin d2n until @wojpawlik fixes his setup to prevent publishing broken versions

**Comment by wojpawlik** [comment]
I disabled auto updates until deno-bin is fixed.

**Comment by roj1512** [comment]
@KnorpelSenf We're passing.

**Review by KnorpelSenf — APPROVED** [review]
TYSM!

**Pull request merged**